content/python/concepts/built-in-functions/terms/any/any.md

```diff
@@ -18,50 +18,48 @@ The `any()` built-in function takes in an iterable object such as a list or tupl
 
 ## Syntax
 
-```py
+```pseudo
 any(iterable)
 ```
 
-## Example 1
+The `iterable` is any sequence or collection that can be traversed, such as a [dictionary](https://www.codecademy.com/resources/docs/python/dictionaries) or [list](https://www.codecademy.com/resources/docs/python/list).
 
-Use `any()` to check if there is a `True` statement in a list `[True, 0, False]`:
+The inside of the pseudocode can be broken down in the following way:
 
-```python
-print(any([True, 0, False]))
-# Output: True
+```pseudo
+def any(iterable):
+  for element in iterable:
+    if element:
+      return True
+  return False
 ```
 
-## Example 2
+Inside the loop, at the first instance of an `element` existing in the `iterable`, execution will stop and `True` will be returned. If the end of the loop is reached and the element is not found, `False` is returned instead.
 
-Use `any()` to check if there is a `True` statement in a dictionary:
+## Example
 
-```python
-my_dict = {1 : "On"}
+Use `any()` to check if there is a `True` statement in a list `[True, 0, False]`:
```

Comment:
Why repeat only one of the list examples here?

Reply:
Do we mean why mention/write out the list but not the dictionary? What do we think of this possible edit:

"
In the example below, the `any()` function is used to return a boolean after checking a list and a dictionary:
"

Reply:
That works.